packages/fe-fpm-writer/test/unit/page/__snapshots__/custom.test.ts.snap

```diff
@@ -807,6 +851,48 @@ exports[`CustomPage generateCustomPage: different versions or target folder with
 "
 `;
 
+exports[`CustomPage generateCustomPage: different versions or target folder with older but supported UI5 version, plus optional page id 1`] = `
+Object {
+  "sap.app": Object {
+    "id": "my.test.App",
+  },
+  "sap.ui5": Object {
+    "dependencies": Object {
+      "libs": Object {
+        "sap.fe.templates": Object {},
+      },
+    },
+    "routing": Object {
+      "routes": Array [
+        Object {
+          "name": "TestObjectPage",
+          "pattern": ":?query:",
+          "target": "TestObjectPage",
+        },
+        Object {
+          "name": "DummyPage",
+          "pattern": "RootEntity:?query:",
+          "target": "DummyPage",
+        },
+      ],
+      "targets": Object {
+        "DummyPage": Object {
+          "options": Object {
```

Comment:
id is missing here. Is this an error?

Reply:
Test is for 1.84 manifest - https://github.com/SAP/open-ux-tools/blob/main/packages/fe-fpm-writer/templates/page/custom/1.84/manifest.json template, it doesn't use `id` property.